quic/s2n-quic-tls-default/Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "Apache-2.0"
 exclude = ["corpus.tar.gz"]
 
 [target.'cfg(unix)'.dependencies]
-s2n-quic-tls = { version = "=0.18.1", path = "../s2n-quic-tls" }
+s2n-quic-tls = { version = "=0.18.2", path = "../s2n-quic-tls" }
 
 [target.'cfg(not(unix))'.dependencies]
 s2n-quic-rustls = { version = "=0.18.1", path = "../s2n-quic-rustls" }
```

Comment:
Same here

Reply:
Why these two specifically (s2n-codec and s2n-quic-tls)?

Reply:
If you bump a dependency of a package, you have to bump that package's version as well. So since you changed dependencies in `s2n-quic-rustls`, the `s2n-quic-rustls` needs to be bumped. Which causes a bump on `s2n-quic-tls-default`

Reply:
Oh my bad you meant s2n-quic-rustls and s2n-quic-tls-default, nvm that makes sense.